package.json

```diff
@@ -25,6 +25,7 @@
     "gulp-sourcemaps": "^2.6.4",
     "jexl": "^1.1.4",
     "minimist": "^1.2.0",
+    "modularscale-sass": "^3.0.8",
```

Comment:
What looks like browser rounding errors knock things off the scale a bit, but I think we can live with it. As an expression of intent it works well, and it removes the previously magic numbers we had, and makes the baseline grid more easily configurable.